packages/builders/src/messages/Message.ts

```diff
@@ -271,7 +273,7 @@ export class MessageBuilder implements JSONEncodable<RESTPostAPIChannelMessageJS
 	public addComponents(
```

Comment:
```suggestion
	public addActionRowComponents(
```

and then needs more methods for the other top-level component types